psi4/src/psi4/dfmp2/mp2.cc

```diff
@@ -1167,7 +1167,6 @@ void RDFMP2::form_Pab() {
     if (max_i <= 0) {
         throw PSIEXCEPTION("Not enough memory in DFMP2");
     }
-    max_i = (max_i <= 0 ? 1 : max_i);
```

Comment:
everyone agree that this line is unneeded after the `throw` above, or am I missing something?

Reply:
I agree!

Reply:
Line 1170 doesn't appear here as it does in the code. Reviewers should take a look at the file to see what's going on.

Reply:
Can you explain what you mean by that? "It doesn't appear here as it does in the code" Are changes not checked in?